Code/src/main/java/nl/utwente/viskell/haskell/env/HaskellCatalog.java

```diff
@@ -144,12 +144,13 @@ public final Environment asEnvironment() {
 
             String name = attributes.getNamedItem("name").getTextContent();
             String signature = attributes.getNamedItem("signature").getTextContent();
+            Boolean isConstructor = attributes.getNamedItem("isConstructor") != null;
```

Comment:
This will probably give confusing results if `<function ... isConstructor="false" />` ends up in the catalog.